components/lcd/esp_lcd_st7796/esp_lcd_st7796.c

```diff
@@ -1,353 +1,36 @@
 /*
- * SPDX-FileCopyrightText: 2023 Espressif Systems (Shanghai) CO LTD
+ * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include <stdlib.h>
-#include <sys/cdefs.h>
-#include "freertos/FreeRTOS.h"
-#include "freertos/task.h"
-#include "esp_lcd_panel_interface.h"
-#include "esp_lcd_panel_io.h"
-#include "esp_lcd_panel_vendor.h"
-#include "esp_lcd_panel_ops.h"
-#include "esp_lcd_panel_commands.h"
-#include "driver/gpio.h"
-#include "esp_log.h"
+#include "soc/soc_caps.h"
 #include "esp_check.h"
+#include "esp_lcd_types.h"
 
 #include "esp_lcd_st7796.h"
+#include "esp_lcd_st7796_interface.h"
 
-static const char *TAG = "st7796";
+const char *TAG = "st7796";
```

Comment:
why changed the visibility?

Reply:
Sorry, I accidentally deleted this. I will fix it later.